guide/installation-procedure.md

```diff
@@ -47,11 +47,10 @@ The advantage of this solution is that Webots will be updated with the system
 updates. This installation requires the `root` privileges.
 
 First of all, you may want to configure your APT package manager by adding the Cyberbotics repository.
-You can easily add it from the `Software and Updates` application.
-In the `Other Software` tab, click on the `Add...` button and copy the following line:
+Simply execute the following line:
```

Comment:
Note that not all Webots users that work in Ubuntu are familiar with the terminal commands.
I'm not completely sure that it is good to remove the graphical procedure to add the Cyberbotics repository even if the terminal command is much simpler.